### PR TITLE
[REF-6128, REF-5911] mlpiper: add support for RestModelServing pipelines

### PR DESCRIPTION
**Pull request by zohar-mizrahi**

### CODE REVIEW

File: mlcomp/parallelm/components/restful/uwsgi_broker.py [hunk 1]
```diff
@@ -224,7 +227,9 @@ def register_model_load_handler(cls, model_path, logger, within_uwsgi_context):
         logger.info("Model file path: {}".format(model_path))
 
         if within_uwsgi_context:
-            cls._model_selector = ModelSelector(model_path)
+            logger.info("{}".format(100 * 'a'))
```
Comment (yakov-g) [inline mlcomp/parallelm/components/restful/uwsgi_broker.py]: Are these debug messages? Please clean.

File: mlcomp/setup.py [hunk 2]
```diff
@@ -10,7 +10,8 @@
 # The text of the README file
 README = open(ROOT + "/README.md").read()
 
-install_requires = ['ml-ops', 'termcolor', 'flask', 'flask_cors', 'psutil', 'py4j', 'nbformat', 'nbconvert']
+install_requires = ['ml-ops', 'termcolor', 'flask', 'flask_cors', 'psutil', 'py4j', 'nbformat', 'nbconvert',
+                    'uwsgidecorators']
```
Comment (yakov-g) [inline mlcomp/setup.py]: now we need to update mcenter_runner and reflex-algos builder containers?

**Review by yakov-g — APPROVED** [review]

File: mlcomp/parallelm/components/restful/constants.py [hunk 3]
```diff
@@ -1,10 +1,13 @@
+import platform
```
Comment (yakov-g) [inline mlcomp/parallelm/components/restful/constants.py]: Please add a comment to CHANGELOG

File: mlcomp/setup.py (hunk 2 above)
Comment (zohar-mizrahi) [reply mlcomp/setup.py]: For reflex-algos - yes, assuming we'll add unit-test for uwsgi with mlpiper, which would run during build. As for the mcenter_runner - it's already installed.

File: mlcomp/parallelm/components/restful/constants.py (hunk 3 above)
Comment (zohar-mizrahi) [reply mlcomp/parallelm/components/restful/constants.py]: What do you mean?

File: mlcomp/parallelm/components/restful/uwsgi_broker.py (hunk 1 above)
Comment (zohar-mizrahi) [reply mlcomp/parallelm/components/restful/uwsgi_broker.py]: Should be removed! Will do ...

**Review by liororama — APPROVED** [review]

**Pull request closed without merging**